src/components/dialog/dialog-actions/dialog-actions.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { Decorator, Meta, StoryObj } from '@storybook/web-components';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+
+import './dialog-actions';
+import readme from './readme.md?raw';
+
+import '../../button';
+import '../../link';
```

Comment:
Not mandatory, but i would prefer to have the specific component imported instead than every button/link..